src/commands/open.ts

```diff
@@ -27,6 +27,10 @@ const LINKS: Link[] = [
   { name: "Database: Rules", arg: "database:rules", consolePath: "/database/rules" },
   { name: "Docs", arg: "docs", url: "https://firebase.google.com/docs" },
   { name: "Dynamic Links", arg: "links", consolePath: "/durablelinks" },
+  { name: "Firestore: Data", arg: "firestore", consolePath: "/firestore/data" },
+  { name: "Firestore: Rules", arg: "firestore:rules", consolePath: "/firestore/rules" },
+  { name: "Firestore: Indexes", arg: "firestore:indexes", consolePath: "/firestore/indexes" },
```

Comment:
Not sure how useful this one would be, considering that we can check the current deployed indexes using the `firestore:indexes` command.